manifests/service.pp

```diff
@@ -18,10 +18,16 @@
     service_content => template('pulpcore/pulpcore-content.service.erb'),
   }
 
-  systemd::unit_file { 'pulpcore-resource-manager.service':
-    content => template('pulpcore/pulpcore-resource-manager.service.erb'),
-    active  => $pulpcore::service_ensure,
-    enable  => $pulpcore::service_enable,
+  if $pulpcore::use_rq_tasking_system {
+    systemd::unit_file { 'pulpcore-resource-manager.service':
+      content => template('pulpcore/pulpcore-resource-manager.service.erb'),
+      active  => $pulpcore::service_ensure,
+      enable  => $pulpcore::service_enable,
+    }
+  } else {
+    systemd::unit_file { 'pulpcore-resource-manager.service':
+      ensure => 'absent',
```

Comment:
I think you should also add `active` and `enable` to make sure they're stopped.

Reply:
Thanks!

I think I initially misread https://github.com/voxpupuli/puppet-systemd/blob/97dd16fa32886b5b0f77a6f38a4953d4c1511c0e/manifests/unit_file.pp#L90-L115 as implying that `ensure => absent` was sufficient